assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -387,6 +395,11 @@ export default Component.extend({
         return;
       }
 
+      if (!isTesting()) {
+        this._observeMessages(this.messages);
```

Comment:
What is the reason we take it out of testing?

Reply:
My bad. Only the last read update should be disabled.